Gosu/Bitmap.hpp

```diff
@@ -88,6 +91,8 @@ namespace Gosu
         }
     };
     
+    void enable_flip_y(bool flip_y);
+    void load_image_inverse_color(Bitmap& bitmap, const std::string& filename);
```

Comment:
These two features seem too obscure to add them to Gosu, they'd be great use cases for a general-purpose shader solution though.